db/fake_data.js

```diff
@@ -0,0 +1,51 @@
+const faker = require('faker');
+const db = require('./sequelize');
+
+const listingData = [];
+const availabilityData = [];
+
+function formatDate(date) {
+  const mm = date.getMonth() + 1;
+  const dd = date.getDate();
+
+  return [date.getFullYear(),
+    (mm > 9 ? '' : '0') + mm,
+    (dd > 9 ? '' : '0') + dd,
+  ].join('-');
+}
+
+for (let i = 0; i < 101; i += 1) {
+  const listing = {
+    price: faker.random.number({ min: 30, max: 1000 }),
+    num_reviews: faker.random.number({ min: 0, max: 300 }),
+    avg_rating: faker.random.number({ min: 0, max: 50 }),
+    service_fee_pct: faker.random.number({ min: 105, max: 110 }),
+    min_nights: faker.random.number({ min: 2, max: 30 }),
```

Comment:
```suggestion
    min_nights: faker.random.number({ min: 2, max: 6 }),
```
Maybe a maximum of 6 for the minimum number or nights? 30 seems aggressive to me.

Reply:
I'm covering short-term rentals as well!